app/views/tokens/_new.html.erb

```diff
@@ -0,0 +1,15 @@
+<div data-controller="modal" class="h-screen w-full fixed flex items-center justify-center top-0 left-0">
```

Comment:
Can we dim the background as well?

Reply:
I can't see a corresponding `modal_controller.js` - is that missing?

Reply:
I didn't originally need it, but I've just added the ability to close the modal without creating a token in cbc3723 and I added it then. Thanks.

I'm also dimming the background in cbc3723